db/migrate/20190708142609_create_conversations_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateConversationsUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :conversations_users do |t|
+      t.references :user, foreign_key: true
+      t.references :conversation, foreign_key: true
```

Comment:
same here, null false?

Reply:
Also, you should add an index on `user` cause you are searching by user above, maybe a multiple index for user and conversation 🤔

Reply:
I think `foreign_key `adds the index.